packages/synapse-react-client/src/components/Markdown/MarkdownEditor.tsx

```diff
@@ -9,10 +9,7 @@ import MarkdownSynapse from './MarkdownSynapse'
 import { UserMentionModal } from './UserMentionModal'
 import { startCase } from 'lodash-es'
 
-export enum MarkdownEditorTabs {
-  WRITE = 'WRITE',
-  PREVIEW = 'PREVIEW',
-}
+const MarkdownEditorTabs = ['WRITE', 'PREVIEW'] as const
```

Comment:
Due to how TS handles string enums, you can't get accurate types when you key into them. For this reason, I changed most of our simple enums like this to static arrays.